src/Common/src/System/IO/PersistedFiles.Unix.cs

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Diagnostics;
+
+namespace System.IO
+{
+    internal static class PersistedFiles
+    {
+        // If we ever need system persisted data, /etc/dotnet/corefx/
+        private const string TopLevelDirectory = "dotnet";
+        // User persisted data, ~/.dotnet/corefx/
+        private const string TopLevelUserDirectory = "." + TopLevelDirectory;
+        private const string SecondLevelDirectory = "corefx";
+
+        // Do not use the user variables in the static initializer in case $HOME didn't resolve.
```

Comment:
Yeah, that was the implicit remainder of that comment.  But since I've removed the field it's a bit less of a trap now.